drizzlepac/runastrodriz.py

```diff
@@ -473,7 +479,7 @@ def process(inFile, force=False, newpath=None, num_cores=None, inmemory=True,
                 updatewcs.updatewcs(_calfiles, checkfiles=False)
                 if _calfiles_flc:
                     updatewcs.updatewcs(_calfiles_flc, checkfiles=False)
-
+            """
```

Comment:
Was this meant to be removed?  If not, please add a comment as to why the code is commented out at this time vs being removed.

Reply:
I still needed to insure that the changes to the code (using 'checkfiles=False' when updatewcs is first called) will remove the need for this code.  This code will be removed if testing with an appropriate dataset indicates there are no problems.  Problems with the testing will then come back to this code which may need to be uncommented again.

Reply:
This commented out code has been removed in ea92282.